corehq/apps/geospatial/tasks.py

```diff
@@ -1,5 +1,22 @@
+from corehq.util.decorators import serial_task
+
 from corehq.apps.celery import task
-from corehq.apps.geospatial.utils import CeleryTaskTracker, update_cases_owner
+from corehq.apps.geospatial.const import INDEX_ES_TASK_HELPER_BASE_KEY
+from corehq.apps.geospatial.utils import (
+    get_celery_task_tracker,
+    CeleryTaskTracker,
+    update_cases_owner,
+    get_geo_case_property,
+)
+from corehq.apps.geospatial.management.commands.index_geolocation_case_properties import (
+    _es_case_query,
```

Comment:
Good point. I have always been a little unsure as to how strict we should be when it comes to marking functions in Python as private. Is having simply a single external reference enough justification, or do we need to start referencing it a few times before it becomes obvious that this is clearly a public function (I have seen plenty of examples in HQ that use both examples)? I suppose this is the potential downside of having it more as a guideline than a rule, it can be difficult to gauge where the line lies at times.

Thinking this through however, the former does make more sense since even a single external reference clearly means its not private/self-contained anymore. Given this, I agree it would make sense to drop the _ here.